docs/userguide/expectations.md

```diff
@@ -471,3 +471,55 @@ end;
 ```
 Since NULL is neither *true* nor *not true*, both expectations will report failure.
 
+# Provide a custom error message
+Expectations allow you to provide a custom error message as second argument:
+````sql
+ut.expect( a_actual {data-type}, 'custom error message if expectation fails' ).to_{matcher};
```

Comment:
Can you replace the 'error message' with 'failure message' or just 'custom message'.
There is a difference between error and failure as we define errored test as a test procedure that raised an unhandled exception.